Source/KeyHandler.mm

```diff
@@ -607,6 +611,22 @@ - (BOOL)handleInput:(KeyHandlerInput *)input state:(InputState *)inState stateCa
         }
     }
 
+    NSLog(@"char code %d", charCode);
+
+    if ([input isControlHold] && (charCode == 28)) {
```

Comment:
Perhaps comment which key combo this is. The PR says the activation key is `Ctrl + .`, but 28 is actually ASCII `FS` and the key combo is (on my keyboard) `Ctrl + \`…